[2024-08-24T18:11:50.063+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-24T18:11:50.100+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_employee.create_output_directory manual__2024-08-24T18:11:48.036128+00:00 [queued]>
[2024-08-24T18:11:50.122+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_employee.create_output_directory manual__2024-08-24T18:11:48.036128+00:00 [queued]>
[2024-08-24T18:11:50.123+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-24T18:11:50.152+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): create_output_directory> on 2024-08-24 18:11:48.036128+00:00
[2024-08-24T18:11:50.171+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'process_employee', 'create_output_directory', 'manual__2024-08-24T18:11:48.036128+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/employee_DAG.py', '--cfg-path', '/tmp/tmpx6himlxu']
[2024-08-24T18:11:50.174+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=516) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-24T18:11:50.180+0000] {standard_task_runner.py:105} INFO - Job 74: Subtask create_output_directory
[2024-08-24T18:11:50.181+0000] {standard_task_runner.py:72} INFO - Started process 517 to run task
[2024-08-24T18:11:50.390+0000] {task_command.py:467} INFO - Running <TaskInstance: process_employee.create_output_directory manual__2024-08-24T18:11:48.036128+00:00 [running]> on host 67412decceee
[2024-08-24T18:11:50.634+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_employee' AIRFLOW_CTX_TASK_ID='create_output_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-08-24T18:11:48.036128+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-24T18:11:48.036128+00:00'
[2024-08-24T18:11:50.640+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-24T18:11:50.655+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-24T18:11:50.697+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-24T18:11:50.698+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=process_employee, task_id=create_output_directory, run_id=manual__2024-08-24T18:11:48.036128+00:00, execution_date=20240824T181148, start_date=20240824T181150, end_date=20240824T181150
[2024-08-24T18:11:50.768+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-24T18:11:50.816+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-24T18:11:50.820+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
